manifests/service.pp

```diff
@@ -6,15 +6,12 @@
 class promtail::service {
   case $facts['kernel'] {
     'Linux': {
+      include systemd::systemctl::daemon_reload
```

Comment:
this resource is gone in latest releases of the systemd module. The module now requires puppet 6.1 or newer because that version contains the automatic systemctl daemon-reload und changed/new unit files.

Reply:
This module already states 6.1 as the bottom end so I think removing this would make sense.